spring-boot-cli/src/test/resources/securityConfiguration.groovy

```diff
@@ -0,0 +1,12 @@
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+
+@Configuration
+class SecurityConfiguration extends WebSecurityConfigurerAdapter {
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http.authorizeRequests()
+                .antMatchers("/shutdown").permitAll()
+    }
+}
```

Comment:
It's because of the oauth2 sample. It enables authorization server and all endpoints become secured. As part of this PR I added the shutdown endpoint which lets me shut the server down gracefully. In the case of the oauth2 sample, I just need the same endpoint but without any security constraints.